src/components/profile-page/profile-page-components/profile-page-item.tsx

```diff
@@ -114,7 +114,7 @@ export const ItemGroupComp = ({
           {(provided) => (
             <div {...provided.droppableProps} ref={provided.innerRef}>
               {itemState.map((item, index) => (
-                <>
+                <div key={index}>
```

Comment:
This is actually worse than using a fragment, since if the order would change it might mess up the rendering.

Reply:
In that case we need something else as a key

Reply:
Didn't know that. But we still need to fix it since an error/warning pops up in the console. What else could we use as key?

Reply:
```
<Fragment key={index}>
  ...
</Fragment>
```
?